webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/xml/GitFindUtils.scala

```diff
@@ -127,6 +130,18 @@ object GitFindUtils extends NamedZioLogger {
     }
   }
 
+  /**
+   * Retrieve revisions for the given path
+   */
+  def findRevFromPath(git: Git, path: String): IOResult[Iterable[RevisionInfo]] = {
+    import scala.jdk.CollectionConverters._
+    IOResult.effectM("") {
```

Comment:
I think you want an error message here :)

Reply:
it was added in the lastest commits